patch/kernel/rockchip-rk3588-edge/overlay/Makefile

```diff
@@ -1,5 +1,6 @@
 # SPDX-License-Identifier: GPL-2.0
 dtbo-$(CONFIG_ARCH_ROCKCHIP) += \
+	rockchip-rk3588-sata1.dtbo \
```

Comment:
I think this Makefile should not be necessary.

Reply:
The auto patcher currently relies on it.

Reply:
Both statements are true.
That makefile should be at least templated (so doesnt need to change to add a dtbo), if not fully _generated_ by the build system. One day.
For now though, it is necessary.

Reply:
> Both statements are true. That makefile should be at least templated (so doesnt need to change to add a dtbo), if not fully _generated_ by the build system. One day. For now though, it is necessary.

So just dtb makefile is auto-generated currently, right?